Assets/Scripts/switchScene.cs

```diff
@@ -9,10 +9,18 @@ public class switchScene : Interact {
 
     public override void interact() {
 		if (isOpen) {
-			SceneManager.LoadScene (nextScene);
+			StartCoroutine ("changeNextScene");
 		}
     }
 
+	IEnumerator changeNextScene(){
+		MechanicAudioManager.getInstance ().playUnlockDoorSound ();
+
+		yield return new WaitForSeconds(MechanicAudioManager.getInstance().unlockDoorClip.length*0.5f);
+
+		SceneManager.LoadScene (nextScene);
```

Comment:
@mahtaia yes, coroutine execute that line after the yield is done
@jayfriso yea I think the clip goes silent, right now 0.5 sounds the best